API.md

```diff
@@ -1,8 +1,52 @@
 Siad API
 ========
 
-Unless otherwise specified, API calls return the JSON object { "Success": true }.
-Errors are sent as plaintext, accompanied by an appropriate status code.
+All API calls return JSON objects. If there is an error, the error is returned
+in plaintext. The standard response is { "Success": true}. In this document,
+the API responses are defined as go structs. The go structs will get encoded to
+JSON before being sent. Go structs are used here to provide type information.
+
+Daemon
+------
+
+Queries:
+
+* /daemon/stop
+* /daemon/update/check
+* /daemon/update/apply
+
+#### /daemon/stop
+
+Function: cleanly shuts down the daemon. May take a while
```

Comment:
what if they are only one sentence? Should they all be full sentences (capital letter at the start?)